src/aind_metadata_service/sharepoint/utils.py

```diff
@@ -0,0 +1,26 @@
+"""Helper methods to map sharepoint fields to schema format"""
+import re
+import string
+from typing import Optional
+
+from aind_data_schema.procedures import Side
+
+
+def map_hemisphere(response_hemisphere) -> Optional[Side]:
+    """Maps response string to Side object"""
+    if response_hemisphere == Side.LEFT.value:
+        return Side.LEFT
+    else:
+        return Side.RIGHT
```

Comment:
Not sure we should assume the if it's not Left that it's Right.

Reply:
good point. in the request form, the field for hemisphere is a choice between Right/Left but I guess if neither is selected itd be "Select ...", i'll update and add another example to cover that case in unit test

Reply:
I think you make it None in that case

Reply:
yup thats how i handled it in the latest commit